src/views/Joseki/Joseki.tsx

```diff
@@ -145,12 +145,7 @@ const ColorMap = {
 
 type MoveType = "bad" | "good" | "computer" | "complete";
 
-interface JosekiProps {
-    match: {
-        params: any;
-    };
-    location: any;
```

Comment:
`props.location` is used down below, I think we need to extend `RouteComponentProps<{ pos: string }>` to add that in right?